src/run/run.c

```diff
@@ -1129,6 +1139,15 @@ static int start_transient_service(
                                 char ts[FORMAT_TIMESPAN_MAX];
                                 log_info("CPU time consumed: %s", format_timespan(ts, sizeof(ts), (c.cpu_usage_nsec + NSEC_PER_USEC - 1) / NSEC_PER_USEC, USEC_PER_MSEC));
                         }
+
+                        if (c.ip_ingress_bytes > 0 && c.ip_ingress_bytes != UINT64_MAX) {
```

Comment:
Hmm, why check for `c.ip_ingress_bytes > 0`? The case where there's accounting on and we know that no bytes were sent/received is also interesting and quite different from the case of no data.